EDK-II-Development-Process.md

```diff
@@ -1,297 +1,222 @@
-First check out [[Getting Started with EDK II]] for downloading the
-latest EDK II development project with your build environment.
+# EDK II Development Process
 
-Are you new to using git? If so, then the [[New to git]] page may be
-helpful.
+First check out [[Getting Started with EDK II]] for downloading the latest EDK II development project with your build
+environment.
 
-The developer process for the EDK II project
---------------------------------------------
+Are you new to using git? If so, then the [[New to git]] page may be helpful.
 
-1.  Setup the EDK II tree if you do not have one
+> **Note:** Commands that you should directly type into a terminal are preceded with `>$`. Unless otherwise noted,
+> these commands are OS agnostic.
 
-    * This is document on the [[SourceForge to Github Quick Start]] page
+If you are new to GitHub or looking for ways to make you GitHub workflow more efficient, please read
+[GitHub & PR Tips](GitHub-PR-Tips.md "wikilink").
 
-2.  Create and checkout a topic branch for new feature or bug fix
+## EDK II Developer Onboarding
 
-    `$ git checkout -b <new-dev-branch> origin/master`
+At a high-level, getting started with code development in the EDK II repo consists of the following activities:
 
-3.  Make changes in the working tree
+1. Tool Setup - Performed once per development machine.
+2. Workspace Setup - Performed once per development workspace.
+3. Development and Test - Performed on every code contribution.
+4. Code Review and CI - Performed on every code contribution.
 
-4.  Break up working tree changes into independent commits that do not
-    break *git bisect*
-    -   [Commit-Partitioning](Commit-Partitioning "wikilink")
+## Tool Setup, Workspace Setup, and Development and Test
 
-    -  To stage all modifications: `$ git add -u`
+Refer to the [How to Build with Stuart](How-to-Build-With-Stuart.md "wikilink") documentation. After following those

```

Comment:
Likely drop this. Mention Stuart in relation to a PR failing and reproducing the issue locally. Will talk about a Stuart led dev process after PR transition.

Reply:
Done